sys/linux/filesystem_amd64.const

```diff
@@ -55,11 +55,11 @@ MS_UNBINDABLE = 131072
 OPEN_TREE_CLOEXEC = 524288
 OPEN_TREE_CLONE = 1
 UMOUNT_NOFOLLOW = 8
-__NR_fsconfig = 338
-__NR_fsmount = 339
-__NR_fsopen = 337
-__NR_fspick = 340
+__NR_fsconfig = 431
+__NR_fsmount = 432
+__NR_fsopen = 430
+__NR_fspick = 433
 __NR_mount = 165
-__NR_move_mount = 336
-__NR_open_tree = 335
+__NR_move_mount = 429
+__NR_open_tree = 428
```

Comment:
@dvyukov Is these change wrong? I didn't modify anythings in filesystem.txt, but it still appear after i run make extrace/generate.

Reply:
I don't know. What are the current syscall numbers for these syscalls? Have they changed recently?

Reply:
The last commit is 16c44697. I try to checkout to 16c44697, and run extract with Linus tree. Get the same change of these files.

Reply:
I also see the same diff. So I guess the values have changed. So your change is good.

Reply:
It seems there is nothing wrong in the whole PR.